invenio_previewer/__init__.py

```diff
@@ -201,13 +210,13 @@
   You can also configure the maximum file size in order to avoid client and
   server freezes. By default it is set to 1MB.
 
-- ``CSV`` - Previews `CSV` files but it can actually works
+- ``CSV`` - Previews `CSV` files but it can actually work
```

Comment:
Here and in all bundled previewers, link to the library integrated library? WDYT?